docs/conf.py

```diff
@@ -36,6 +37,10 @@
     "rasabaster.editlink",
     "rasabaster.runnable",
     "rasabaster.conversations",
+    "rasabaster.ace",
+    "rasabaster.train_button",
+    "rasabaster.download_button",
+    "rasabaster.chat_block",
 ]
```

Comment:
@m-vdb will you add these to the extensions list for the docs switch?

Reply:
I've created a new ticket for @alwx to port these parts to the new docs :+1: https://github.com/RasaHQ/rasa/issues/6269

Reply:
Awesome thanks 🙏